Alcatraz/Controllers/ATZPluginWindowController.m

```diff
@@ -255,12 +255,22 @@ - (void)displayScreenshotForPackage:(ATZPackage *)package {
     self.previewPanel.title = package.name;
 
     [self.tableViewDelegate fetchAndCacheImageForPackage:package progress:NULL completion:^(NSImage *image) {
-        [self displayImage:image withTitle:package.name];
+        [self displayImage:image package:package];
     }];
 }
 
-- (void)displayImage:(NSImage *)image withTitle:(NSString*)title {
+- (void)displayImage:(NSImage *)image package:(ATZPackage*)package {
+    if ([package.screenshotPath hasSuffix:@".gif"]) {
```

Comment:
I'm a little uneasy relying on the file extension to identify gifs. I'd prefer we look into the image data to be sure the file is indeed an animated gif.
Something like https://gist.github.com/keefo/5344890 `isGifImage`.